<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDcmcWay1Widget</class>
 <widget class="QWidget" name="CDcmcWay1Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{border-image: url(:/mul/dcmc/first.png);}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="mRg1WrVal">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>72</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mBd1OutVal">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>538</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg2Val">
    <property name="geometry">
     <rect>
      <x>517</x>
      <y>98</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1IncVal">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>213</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg1WrHint">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>68</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3RstVal">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>435</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1SelHint">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>178</y>
      <width>40</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1Out3Hint">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>257</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(p9)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mBd1OutHint">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>532</y>
      <width>40</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mSm1OutVal">
    <property name="geometry">
     <rect>
      <x>236</x>
      <y>383</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mRg2WrHint">
    <property name="geometry">
     <rect>
      <x>653</x>
      <y>68</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3WrHint">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>445</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg2WrVal">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>75</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg1ShrHint">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>88</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3WrVal">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>451</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mDVal">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>5</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1SelVal">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>184</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3RstHint">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>429</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mBd2OutHint">
    <property name="geometry">
     <rect>
      <x>607</x>
      <y>532</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mBd2Val">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>559</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1Out3Val">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>269</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1WrHint">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>188</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1Val">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>220</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mRg1ShrVal">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>93</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg1Out0Hint">
    <property name="geometry">
     <rect>
      <x>407</x>
      <y>141</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(p9)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg1Out0Val">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>152</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mBd1Val">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>558</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3Val">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>463</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1WrVal">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>195</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mCt1IncHint">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>208</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg1Val">
    <property name="geometry">
     <rect>
      <x>293</x>
      <y>98</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mBd2OutVal">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>538</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mXor1Val">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>277</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1NotVal">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>254</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1NotHint">
    <property name="geometry">
     <rect>
      <x>527</x>
      <y>248</y>
      <width>40</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3ShrVal">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>467</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg3ShrHint">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>461</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg4Val">
    <property name="geometry">
     <rect>
      <x>473</x>
      <y>464</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mRg4ShrVal">
    <property name="geometry">
     <rect>
      <x>658</x>
      <y>441</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg4ShrHint">
    <property name="geometry">
     <rect>
      <x>631</x>
      <y>435</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mSm1CHint">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>313</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(y99)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mSm1CVal">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>319</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg1Out7Hint">
    <property name="geometry">
     <rect>
      <x>311</x>
      <y>129</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(p9)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg1Out7Val">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg2Out7Hint">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>129</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">(p9)</string>
    </property>
   </widget>
   <widget class="QLabel" name="mRg2Out7Val">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">0</string>
    </property>
   </widget>
   <widget class="QLabel" name="mMs1Val">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>205</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;</string>
    </property>
    <property name="text">
     <string notr="true">00000000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
